doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -553,64 +560,6 @@ with corresponding nitrogen fluxes:
 
 where CF is the carbon flux, CS is stored carbon, NF is the nitrogen flux, NS is stored nitrogen, and :math:`biofuel\_harvfrac` is the harvested fraction of leaf/livestem for biofuel feedstocks.
 
-.. _Table Plant functional type (PFT) parameters for harvested fraction of leaf/livestem for bioenergy production:
```

Comment:
@samsrabin I'm just not sure why this table is being removed?

Reply:
It's an inefficient use of space. The information is now more succinctly conveyed at line 517:
> For the biofuel crops Miscanthus and switchgrass, 70% of live leaf and stem biomass at harvest is transferred to the crop product pool as described for "food" harvest above. This value can be changed for these crops—or set to something other than the default zero for any other crop—with the parameter `biofuel_harvfrac` (0-1).

Reply:
Oh, you are right there is a table where everything other than the last two are zeros! No need for that.